### ISSUE TITLE
Crash: Game crashes when building bed

### ISSUE THREAD

**Issue by apcnc** [CompletelyFairGames/dwarfcorp#911]
https://mega.nz/#F!SM12ASRK!Uvt1vNoRmS0zwQJjRFvT1A

The game crashes roughly 1-2 min after loading the game and doesn't crash when i cancel the build orders.
Screenshot was taken a second before the game crashed.
I'm playing on the current steam build.

**Comment by com1clyf3** [comment]
@apcnc  Did these crashes create a crashlog or two? They should be individual files found specifically in "logging". You've sent us your "log" which is helpful, but not the same thing.

**Comment by apcnc** [comment]
@com1clyf3 No, it didn't create one

**Comment by mklingen** [comment]
Thank you for the report,  I managed to load your game (it's insane by the way, that's a lot of dwarfs). However I wasn't able to reproduce the crash. Did the game say anything upon crashing? Was there a popup?

**Comment by com1clyf3** [comment]
@apcnc ^

**Comment by apcnc** [comment]
@mklingen no, it didn't say anything. And i'm also semi-sure that it has something to do with the performance since this only happens when the game starts to slightly lag and also now happens when i build other things the dwarfs interact with like chairs/barrels
